charts/datadog/templates/_helpers.tpl

```diff
@@ -669,6 +669,13 @@ securityContext:
 {{- else }}
 securityContext:
 {{ toYaml .securityContext | indent 2 }}
+{{- if and .seccomp .kubeversion (semverCompare ">=1.19.0" .kubeversion) }}
+  seccompProfile:
+    type: {{ if hasPrefix "localhost/" .seccomp }}Localhost{{ else if eq "runtime/default" .seccomp }}RuntimeDefault{{ else }}Unconfined{{ end }}
+    {{- if hasPrefix "localhost/" .seccomp }}
+    localhostProfile: {{ .seccomp }}
+    {{- end }}
```

Comment:
could you try to refactor the code to have the "if,else if,else" on different lignes

you can use `{{- ... -}}` to strip the lines.

Reply:
Updated.